crash/src/app/accident-report-container/map/map.component.ts

```diff
@@ -0,0 +1,188 @@
+
+
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+import { GoogleMap, GoogleMapsModule } from '@angular/google-maps';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { CommonModule} from '@angular/common';
+import { RouterOutlet } from '@angular/router';
+import { CrashService } from '../../crash.service';
+import { ReactiveFormsModule } from '@angular/forms';
+import { provideNativeDateAdapter } from '@angular/material/core';
+import { Loader } from '@googlemaps/js-api-loader';
+import { environment } from '../../../environments/environment';
+
+
+
+@Component({
+  selector: 'crash-map',
+  standalone: true,
+  providers: [provideNativeDateAdapter()],
+  imports: [CommonModule, RouterOutlet, GoogleMapsModule,  MatExpansionModule, ReactiveFormsModule,
+  ],
+  templateUrl: './map.component.html',
+  styleUrl: './map.component.scss'
+})
+
+
+export class MapComponent implements OnInit {
+
+  constructor(private crashservice: CrashService) { }
+
+  loadAPIMapscript() {
+
+    let loader = new Loader({
+      apiKey: environment.GOOGLE_API_KEY,
+      version: "weekly",
+      libraries: ["places"]
+    });
+
+    loader
+      .importLibrary('maps')
+      /*  
+      dynamic loading of map does not work
+      .then(({Map}) => {
+          new google.maps.Map(document.getElementById("map") as HTMLElement,this.options);
+       })  */
+      .catch((e) => {
+        // do something
+        console.log("error caught");
```

Comment:
remove console logs, in the case of an error we can use `throw new Error() instead`